examples/layers/JSONLayers/Region.json

```diff
@@ -7,7 +7,8 @@
 	"style"      : "",
 	"projection" : "EPSG:4326",
 	"transparent" : true,
-	"bbox"      : [-61.80983300000002, -21.38963079935857, 55.83665389999998, 51.0890011990783],
+    "unit"      : 1,
```

Comment:
Well projection covers a lot more thing.
Actually I was about to remove this attribute since we don't use it.
And then, *later*, restore it but to use it completely (using proj4 to transform from 1 proj to another).